NAPS2.Lib/Dependencies/DownloadController.cs

```diff
@@ -11,84 +10,41 @@ public class DownloadController
     private readonly ScanningContext _scanningContext;
     private readonly ILogger _logger;
 
-    // TODO: Migrate to HttpClient
-#pragma warning disable SYSLIB0014
-    private readonly WebClient _client = new();
-#pragma warning restore SYSLIB0014
+    private static readonly HttpClient s_client = new();
```

Comment:
NAPS2 naming convention for static readonly is PascalCase; let's call this `DefaultHttpClient`. Let's also move it to the top of the class (static members first).